core/chains/evm/gas/rollups/op_l1_oracle.go

```diff
@@ -538,3 +525,15 @@ func (o *optimismL1Oracle) getEcotoneFjordGasPrice(ctx context.Context) (*big.In
 
 	return new(big.Int).Div(scaledGasPrice, scale), nil
 }
+
+func encodeCalldata(abiString, methodName string) ([]byte, abi.ABI, error) {
+	methodAbi, err := abi.JSON(strings.NewReader(abiString))
+	if err != nil {
+		return nil, abi.ABI{}, fmt.Errorf("failed to parse ABI: %w", err)
+	}
+	calldata, err := methodAbi.Pack(methodName)
+	if err != nil {
+		return nil, abi.ABI{}, fmt.Errorf("failed to pack calldata: %w", err)
```

Comment:
nit: Could you include the `methodName` in this error?
```suggestion
		return nil, abi.ABI{}, fmt.Errorf("failed to pack calldata for %s: %w", methodName, err)
```